pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -203,10 +211,73 @@ func (o *MustGatherOptions) completeImages() error {
 		}
 		o.Images = append(o.Images, image)
 	}
+	if o.AllImages {
+		// find all csvs and clusteroperators with the annotation "operators.openshift.io/must-gather-image"
+		var csvs []byte
+		var cos []byte
+		var err error
+		var csvPath = "/apis/operators.coreos.com/v1alpha1/clusterserviceversions"
+		var coPath = "/apis/config.openshift.io/v1/clusteroperators"
+		var pluginImages = map[string]struct{}{}
+
+		csvs, err = o.Client.Discovery().RESTClient().Get().
+			AbsPath(csvPath).
+			DoRaw(context.TODO())
```

Comment:
These raw requests look to me quite _hacky_. Can't we filter these resources based on label selectors or field selectors?

Reply:
> These raw requests look to me quite hacky.

Do you suggest using ClientSet instead?

> Can't we filter these resources based on label selectors or field selectors?

We're looking for a specific annotation, and based on my web search-fu, there isn't a way to filter the resources based on annotations. Let me know if you have any ideas.